### PR TITLE
Tech dept send grid razor view engine maior

### PR DESCRIPTION
**Pull request by herakros**
You will need a working e-mail address, you need to check the SendGrid service, to do this, ask the developer ApiKey and paste into the program settings, you can also add a new MailView and try to integrate it.

### CODE REVIEW

File: Provis.WebApi/Views/Mails/WorkspaceInvite.cshtml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<table align="center" style="text-align:center;">
+    <tr>
+        <td>
+            <h1 style="color: white;">Owner: @Model.Owner</h1>
+            <h3 style="color: white;">Welcome to my workspace - @Model.WorkspaceName</h3>
+        </td>
+    </tr>
+</table>
+<div style="text-align:center;">
+    <a href="http://localhost:4200/user/workspaces" style="margin: 20px;">
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Mails/WorkspaceInvite.cshtml]: We should use dynamic address

File: Provis.WebApi/Views/Mails/ConfirmEmail.cshtml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿@model string
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Mails/ConfirmEmail.cshtml]: Let's create an instance of object and do not use just string.

File: Provis.WebApi/Views/Mails/ConfirmEmail.cshtml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+@{
+    Layout = "_MailLayout";
+}
+
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Mails/ConfirmEmail.cshtml:8]: Let's say hello to user name

File: Provis.WebApi/Views/Mails/TwoFactorCode.cshtml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    Layout = "_MailLayout";
+}
+
+<table align="center">
+    <tr>
+        <td>
+            <h1>Your Code:</h1>
+        </td>
+        <td align="right">
+            <h1 style="color: dodgerblue;">@Model</h1>
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Mails/TwoFactorCode.cshtml]: Same here

File: Provis.WebApi/Views/Shared/_MailLayout.cshtml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        </tr>
+        <tr>
+            <td bgcolor="#2E445F" style="padding: 5px 30px 5px 30px">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                    <tr>
+                        <td>
+                            <h2 style="color: white;">&reg; Project Provis 2022<br /></h2>
+                        </td>
+                        <td align="center">
+                            <a href="http://localhost:4200/">
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Shared/_MailLayout.cshtml]: Let's don't use cons url

File: Provis.WebApi/Views/Shared/_MailLayout.cshtml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            <td bgcolor="#6E67A9" style="padding: 40px 0 30px 0;">
+                @RenderBody()
+            </td>
+        </tr>
+        <tr>
+            <td bgcolor="#2E445F" style="padding: 5px 30px 5px 30px">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                    <tr>
+                        <td>
+                            <h2 style="color: white;">&reg; Project Provis 2022<br /></h2>
```
Comment (YevheniiVovk) [inline Provis.WebApi/Views/Shared/_MailLayout.cshtml]: Year is not const too

**Review by YevheniiVovk — APPROVED** [review]

**Review by pashunskyi — APPROVED** [review]

**Pull request merged**